client/components/SongCommentsEntry.jsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class SongCommentsEntry extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { show: false };
+    this.createComments(props);
+    this.showComment = this.showComment.bind(this);
+    this.hideComment = this.hideComment.bind(this);
+  }
+
+  createComments({ comment }) {
+    const max = 22;
+    const { text, artistName, time } = comment;
+    const data = { artist: artistName };
+    data.text = (text.length > max) ? `${text.slice(0, max)}...` : text;
+
+    let words = ['artist', 'text'];
+    this.position = 'player-comment-right';
+    this.comments = [];
+    if (time > 50) {
+      words = ['text', 'artist'];
+      this.position = 'player-comment-left';
+    }
+    words.forEach((word) => {
+      this.comments.push(
+        <span key={word} className={`player-comment-${word}`}>
+          {`${data[word]}    `}
+        </span>,
+      );
+    });
+  }
+
+  showComment() {
+    this.props.resetNowPLaying();
+    this.setState({ show: true });
+  }
+
+  hideComment() {
+    this.setState({ show: false });
+  }
+
+  render() {
+    const { show } = this.state;
+    const { comment, handleReply, nowPlaying } = this.props;
+    const commentDetails = (nowPlaying || show) ? (<p className={`${this.position}`}>{this.comments}</p>) : '';
+
+    return (
+      <div
+        className="player-comment-single"
+        onMouseOver={this.showComment}
+        onMouseLeave={this.hideComment}
```

Comment:
https://javascript.info/mousemove-mouseover-mouseout-mouseenter-mouseleave

Similar to Marlon's example (yours doesn't have a glitch) but it would make sense to do a mouse over/out and mouse enter/leave. I played around with both on your component and both seem to be fine especially because there are no children inside the comment entry. The article I posted is helpful to understand the differences.  Also - double check your eslinter because mine is giving me an error on your code when there is an over/leave.

Reply:
Done